### PR TITLE
Done the widgets import implemented

### PR DESCRIPTION
**Pull request by briansetzesglobant**
The widgets import was implemented from the main repository

Video:
https://user-images.githubusercontent.com/84338449/124657394-d53e3e00-de78-11eb-85eb-c7d8a1a3e774.mp4

### CODE REVIEW

File: lib/src/ui/home_page.dart [hunk 1]
```diff
@@ -99,8 +97,8 @@ class HomePageState extends State<HomePage> {
           AsyncSnapshot<Movie> snapshot,
         ) {
           return snapshot.hasData
-              ? buildGrid(
-                  snapshot.data!,
+              ? PopularPage(
```
Comment (MarcosCGlobant) [inline lib/src/ui/home_page.dart]: can we rename this widget to PopularMoviesGrid?, it's the builder for the Grid and not a page, so the name should reference to the Grid

Comment (briansetzesglobant) [reply lib/src/ui/home_page.dart]: done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

File: lib/src/ui/home_page.dart [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-import '../resources/movie_strings.dart';
-import '../widgets/movie_detail_image.dart';
 import '../../utils/constants.dart';
 import '../../utils/text_styles.dart';
 import '../blocs/i_movie_bloc.dart';
 import '../models/movie.dart';
-import 'home_detail_page.dart';
+import 'popular_movies_grid.dart';
 
 class HomePage extends StatefulWidget {
```
Comment (luisjavierglb) [inline lib/src/ui/home_page.dart:8]: HomePageState should be private to this file

**Review by luisjavierglb — CHANGES_REQUESTED** [review]
Small change requested

File: lib/src/ui/home_page.dart (hunk 2 above)
Comment (briansetzesglobant) [reply lib/src/ui/home_page.dart:8]: done

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**